runtime/oti/j9nonbuilder.h

```diff
@@ -5135,7 +5201,7 @@ typedef struct J9JavaVM {
 	omrthread_monitor_t systemPropertiesMutex;
 	U_8* javaHome;
 	void* cInterpreter;
-	void* bytecodeLoop;
+	void (*bytecodeLoop)(struct J9VMThread *currentThread);
```

Comment:
According to `vm_api.h` [1] those functions have signature `UDATA (*)(J9VMThread *)` while the assignment in `jvminit.c` expects this field to be of type `void *` [2]: I don't expect this to compile without further changes.

[1] https://github.com/eclipse/openj9/blob/master/runtime/oti/vm_api.h#L4069-L4080
[2] https://github.com/eclipse/openj9/blob/master/runtime/vm/jvminit.c#L2414-L2416

Reply:
I changed the existing code as discussed with @keithc-ca,  will compile a cross-build with the latest changes to ensure whether it works fine in the cross-compilation.